src/main/java/org/minions/devfund/angela/katas/MorseCodeDecoder.java

```diff
@@ -0,0 +1,89 @@
+package org.minions.devfund.angela.katas;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+final class MorseCodeDecoder {
+    final static private Map<String, String> MorseCode = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)